src/App.jsx

```diff
@@ -1,104 +1,25 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+const posts = postsFromServer.map(post => {
+  const updatedPost = {
+    ...post,
+    comments:
+      commentsFromServer.filter(comment => comment.postId === post.id) || null,
```

Comment:
The use of `|| null` is unnecessary here. The `filter` method will always return an array, so `comments` will never be `null`. If you want to assign `null` when there are no comments, you should check if the array is empty and then assign `null`.